src/transformers/modeling_flax_bert.py

```diff
@@ -0,0 +1,365 @@
+from typing import Callable, Dict
+
+import numpy as np
+
+import flax.linen as nn
```

Comment:
(nit) `nn` looks very PyTorch like -> do you think we could adapt a different naming here? People might get confused by this I think